src/playbacks/html5_video/html5_video.js

```diff
@@ -269,8 +269,11 @@ export default class HTML5Video extends Playback {
   }
 
   ready() {
-    this.trigger(Events.PLAYBACK_READY, this.name)
+    if (this.isReadyState) {
+      return
```

Comment:
Because it looks like it could be called [here](https://github.com/clappr/clappr/blob/fix-hlsjs-ready-event/src/playbacks/html5_video/html5_video.js#L302) as well as [here](https://github.com/clappr/clappr/blob/fix-hlsjs-ready-event/src/playbacks/hls/hls.js#L128)